manage.sh

```diff
@@ -57,22 +57,27 @@ function install_conda {
     esac
 
     if conda --version ; then
-        echo Conda already installed. Skipping conda installation.
+	if  [ `(conda --version | cut -d' ' -f2)` == "4.8.2" ]; then
+            echo Conda already installed.
+	else
+	    conda activate base
+	    conda install conda=4.8.2
+	fi
     else
         echo Installing conda for $CONDA_OS
         if which wget; then
-            wget https://repo.continuum.io/miniconda/Miniconda3-4.5.4-${CONDA_OS}-x86_64.sh -O miniconda.sh
+            wget https://repo.continuum.io/miniconda/Miniconda3-py${PYTHON_VERSION//.}_4.8.2-${CONDA_OS}-x86_64.sh -O miniconda.sh
         else
-            curl https://repo.continuum.io/miniconda/Miniconda3-4.5.4-${CONDA_OS}-x86_64.sh -o miniconda.sh
+            curl https://repo.continuum.io/miniconda/Miniconda3-py${PYTHON_VERSION//.}_4.8.2-${CONDA_OS}-x86_64.sh -o miniconda.sh
         fi
         bash miniconda.sh -b -p $HOME/miniconda
-        CONDA_SH=$HOME/miniconda/etc/profile.d/conda.sh
-        source $CONDA_SH
-        echo Activated conda by sourcing $CONDA_SH
+	CONDA_SH=$HOME/miniconda/etc/profile.d/conda.sh
+	source $CONDA_SH
+	echo Activated conda by sourcing $CONDA_SH
```

Comment:
why did you dedent? It shouldn't be dedented as it refers to the miniconda autoinstalled.